src/NuGet.Clients/VisualStudio/Extensibility/IVsPackageProjectMetadata.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// contain information about project where we're executing nuget actions
```

Comment:
For the public interfaces all comments should be complete sentences, start with a capital letter, and end with a period. Except that these will be copied directly from the code to MSDN or wherever the nuget API docs will go.